src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryColor.cs

```diff
@@ -62,28 +62,29 @@ private static byte ParseColor(Match match, string color)
 
         public static FormulaValue RGBA(IRContext irContext, NumberValue[] args)
         {
+            // Truncate (strip decimals)
```

Comment:
With #1072, can we now do the truncation at the IR level? 

We'd just need to implement TexlFunction.GetArgPreprocessor - in theory, LibraryColor.cs wouldn't even need any changes.

Reply:
Updated function definition to use truncation at IR